pkg/connector/org_role.go

```diff
@@ -198,22 +188,33 @@ func (o *orgRoleResourceType) Grants(
 		ret = append(ret, grant)
 	}
 
+	// If we got a full page of teams, return them and let the next page handle users
+	if len(teams) == pToken.Size {
+		pageToken, err := bag.NextToken(fmt.Sprintf("%d", page+1))
+		if err != nil {
+			return nil, "", nil, err
+		}
+		return ret, pageToken, nil, nil
+	}
```

Comment:
I'm not sure if we actually set size all the time, and if the token we return is just the page I think it will do the `ListTeamsAssignedToOrgRole` if theres a second page from `ListUsersAssignedToOrgRole`. We usually paginate by pushing a page to the bag when we have multiple actions we want to do within a method like here so we can distinguish which action to do 

See this example: 
https://github.com/ConductorOne/baton-okta/blob/81354f306dc69ed74754499a8ea8bb5b234b6943/pkg/connector/group.go#L124